server/views/pages/confirmDISFormsIssued.njk

```diff
@@ -5,7 +5,7 @@
 {% from '../macros/adjudicationTabHeaderAll.njk' import adjudicationTabHeaderAll %}
 {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
 
-{% set title = "Confirm DIS 1/2 has been issued to prisoner" %}
+{% set title = "Confirm notice of being placed on report was issued" %}
```

Comment:
I did a find and replace on `Confirm DIS 1/2 has been issued to prisoner` and this page should have been updated in the PR https://github.com/ministryofjustice/hmpps-manage-adjudications/pull/853 but no biggie